troposphere/ecs.py

```diff
@@ -26,6 +26,20 @@ class DeploymentConfiguration(AWSProperty):
     }
 
 
+class PlacementConstraint(AWSProperty):
+    props = {
+        'Type': (basestring, True),
+        'Expression': (basestring, False),
+    }
+
+
+class PlacementStrategy(AWSProperty):
+    props = {
+        'Type': (basestring, True),
```

Comment:
I was adding that, but then I noticed that they had no other validators of this type. I could definitely do this, I wanted to see what cloudtools suggests